packages/eas-cli/src/credentials/android/actions/__tests__/CreateGoogleServiceAccountKey-test.ts

```diff
@@ -0,0 +1,29 @@
+import { createCtxMock } from '../../../__tests__/fixtures-context';
+import { getAppLookupParamsFromContextAsync } from '../BuildCredentialsUtils';
+import { CreateGoogleServiceAccountKey } from '../CreateGoogleServiceAccountKey';
+
+jest.mock('../../../utils/promptForCredentials', () => ({
+  getCredentialsFromUserAsync: jest.fn(() => ({
+    keyJson: JSON.stringify({ private_key: 'super secret' }),
+  })),
+}));
+
+describe(CreateGoogleServiceAccountKey, () => {
+  it('creates a Google Service Account Key in Interactive Mode', async () => {
+    const ctx = createCtxMock({ nonInteractive: false });
+    const appLookupParams = await getAppLookupParamsFromContextAsync(ctx);
+    const createGsaKeyAction = new CreateGoogleServiceAccountKey(appLookupParams.account);
+    await createGsaKeyAction.runAsync(ctx);
+
+    // expect fcm api key to be created on expo servers
+    expect(ctx.android.createGoogleServiceAccountKeyAsync as any).toHaveBeenCalledTimes(1);
```

Comment:
I've seen a few other lines like this in your tests. Just curious - why `toHaveBeenCalledTimes(1)` and not `toHaveBeenCalled()`? 🤔 
I'm not asking to change this :P

Reply:
A while back, I accidentally introduced a bug where I was making calls to this more than once, hence the `toHaveBeenCalled(1)` 😓

Reply:
`as any` is not necessary. I just noticed this and changed all tests - https://github.com/expo/eas-cli/commit/1486b481f6233b5b5ae47fdbc9f9086a1ecf4bd6